UseCases.md

```diff
@@ -51,3 +51,16 @@
     - Blogs wants to read data about Alice’s interests
     - Alice does not want Blogs to get data about her interests
     - Blogs continually asks Alice to grant it access to her interests and Alice is annoyed with the incessant asking
+ - Alice uses or is developing http://localhost:8080 to view her photos
+    - Alice's Identity Provider can't reach Alice's `localhost:8080`
+    - Alice's Pod can't reach Alice's `localhost:8080`
+ - Alice uses https://coolcode.int.enterprise.example to edit code stored in Customer Bob's Pod
+    - *CoolCode* is deployed behind Enterprise.Example's company firewall
+      - *CoolCode* is proprietary to Enterprise.Example; or
+      - *CoolCode* is a commercial product that is deployed on-premises at Enterprise.Example's datacenter
+    - Alice's Identity Provider can't reach https://coolcode.int.enterprise.example
+    - Customer Bob's Pod can't reach https://coolcode.int.enterprise.example
```

Comment:
Same thing here. This should be phrased as a use case with a goal that Alice has to accomplish and not just a setup.

Reply:
the intent here was to illustrate by example a possible and legitimate configuration of actors that should work, which is how all the other use cases work. i can rephrase this to more explicitly call out the intent to cover the configuration, and then give the example.

Reply:
note too that this use case already includes an implicit goal similar to all the other use cases: "Alice uses X **to edit code stored in Customer Bob's Pod**"